Orange/widgets/gui.py

```diff
@@ -1701,6 +1701,60 @@ def valueSlider(widget, master, value, box=None, label=None,
     return slider
 
 
+class OrangeComboBox(QtGui.QComboBox):
+    """
+    A QtGui.QComboBox subclass extened to support bounded contents width hint.
+    """
+    def __init__(self, parent=None, maximumContentsLengthHint=-1, **kwargs):
+        super().__init__(parent, **kwargs)
```

Comment:
> I don't think QComboBox supports any arguments besides parent.

PyQt allows (with some caveats) for any QObject's properties to be set with ``__init__`` keyword parameters.